src/main/flight/pid_init.c

```diff
@@ -272,6 +279,7 @@ void pidUpdateFeedforwardLpf(uint16_t filterCutoff)
     if (filterCutoff > 0) {
         for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
             pt3FilterUpdateCutoff(&pidRuntime.feedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
+            pt3FilterUpdateCutoff(&pidRuntime.angleFeedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
```

Comment:
```suggestion
            pt3FilterInit(&pidRuntime.angleFeedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.level_response_gain));
```

Reply:
same comment as above, just don't do this its so hacky. Just raise the cutoff or lower it, don't mess with the dt.